packages/foreman/foreman-fapolicyd/foreman-fapolicyd.spec

```
@@ -11,6 +11,9 @@ Source0:  https://codeload.github.com/theforeman/%{name}/tar.gz/v%{version}#/%{n
 BuildArch: noarch
 
 Requires: fapolicyd
+Requires: systemd
+
+BuildRequires: systemd
```

I think this is sufficient
```suggestion
BuildRequires: systemd-rpm-macros
```